src/indra_cogex/apps/utils.py

```diff
@@ -204,15 +253,24 @@ def format_stmts(
 
     stmt_rows = []
     for stmt in stmts:
-        row = _stmt_to_row(
-            stmt,
-            cur_dict=cur_dict,
-            cur_counts=cur_counts,
-            remove_medscan=remove_medscan,
-            source_counts=source_counts_per_hash.get(stmt.get_hash())
-            if source_counts_per_hash
-            else None,
-        )
+        if isinstance(stmt, list):
+            stmt_hash, evidence = stmt[0], stmt[1]
+            row = _stmt_to_row(
+                stmt_hash,
+                evidence=evidence,
+                cur_dict=cur_dict,
+                cur_counts=cur_counts,
+                remove_medscan=remove_medscan,
+                source_counts=source_counts_per_hash.get(stmt_hash) if source_counts_per_hash else None,
+            )
+        else:
+            row = _stmt_to_row(
+                stmt,
+                cur_dict=cur_dict,
+                cur_counts=cur_counts,
+                remove_medscan=remove_medscan,
+                source_counts=source_counts_per_hash.get(stmt.get_hash()) if source_counts_per_hash else None,
+            )
```

Comment:
Something is off here:
- `_stmt_to_row` takes an indra `Statement` as its first argument, not a statement hash.
- There is no kwarg for `evidence`, yet it is passed in the when the `if` branch of the code is executed.

Reply:
Fixed in cbdb3c61